BoxContentSDK/BoxContentSDK/Requests/BOXRequest.m

```diff
@@ -48,6 +48,15 @@ - (NSString *)baseURL
     return _baseURL;
 }
 
+- (NSString *)baseURLWithoutVersion
```

Comment:
I don't think this needs to be in the Request base class, right? It's not particularly common, we can just use it if/when we have a concrete request subclass that needs it.